source/v2/openapi.yaml

```
@@ -1164,9 +1164,10 @@ paths:
       parameters:
         - name: query[letter]
           in: query
-          required: true
+          required: false
           description: |
             Retrieve search references started with  `query[letter]` letter.
+            If param letter is not provided, this endpoint responses with the list of all the SearchReferences.
```

This reads a bit strangely :thinking: 

```suggestion
            Without a query parameter all search references are returned in the response.
```

---

👍